packages/audio-block/src/components/BlockAttachments.tsx

```diff
@@ -11,7 +11,7 @@ export const BlockAttachments = ({ appBridge }: BlockAttachmentsProps) => {
         onAttachmentDelete,
         onAttachmentReplace,
         onAttachmentsSorted,
-    } = useAttachments(appBridge, ATTACHMENTS_SETTING_ID, ATTACHMENTS_SETTING_ID);
+    } = useAttachments(appBridge, ATTACHMENTS_SETTING_ID);
```

Comment:
```suggestion
    } = useAttachments(appBridge, ATTACHMENTS_ASSET_ID);
```

It isn't a setting ID but an asset ID